libraries/libraries.json

```diff
@@ -528,6 +555,34 @@
     "logoAvailable": "true",
     "ensemble": "false"
   },
+  {
+    "name": "RichTextArea",
+    "summary": "RichTextArea is a text input control which provides rich text features along with emoji, and non-text objects like images, tables and hyperlinks.",
+    "tags": "custom controls, rich text editor",
+    "description": "The RichTextArea control, presented by Gluon, is built with standard Java and JavaFX APIs. It supports rich text, emojis, images, tables, and hyperlinks. ",
+    "documentation": "https://github.com/gluonhq/rich-text-area#readme",
+    "license": "GPLV3",
+    "homepage": "",
+    "javadocs": "https://javadoc.io/doc/com.gluonhq/rich-text-area/latest/com.gluonhq.richtextarea/module-summary.html",
+    "repository": "https://github.com/gluonhq/rich-text-area",
+    "githubAccount": "gluonhq",
+    "githubProject": "rich-text-area",
+    "issueTracker": "https://github.com/gluonhq/rich-text-area/issues",
+    "id": "richtextarea",
+    "companyId": "gluon",
+    "createdOn": "2023-09-06",
+    "modifiedOn": "2023-09-06",
+    "groupId": "com.gluonhq",
+    "artifactId": "rich-text-area",
+    "logoAvailable": "false",
+    "ensemble": "true",
+    "peopleIds": [
+      "j.pereda",
+      "e.ryzhikov",
+      "a.agarwal",
+      "d.lemmermann"
```

Comment:
Please remove me from the people list. I was involved but only on the customer site of this project.